README.md

```diff
@@ -118,13 +119,13 @@ wadm:
 You can deploy Wadm using your values file and Helm:
 
 ```sh
-helm install wadm -f values.yaml --version 0.2.0 oci://ghcr.io/wasmcloud/charts/wadm
+helm install wadm -f wadm-values.yaml --version 0.2.0 oci://ghcr.io/wasmcloud/charts/wadm
 ```
 
 ### Start the operator
 
 ```sh
-kubectl kustomize build deploy/local | kubectl apply -f -
+kubectl kustomize deploy/local | kubectl apply -f -
```

Comment:
I'm not sure if this is intended for running it locally vs. "production", but if it's the latter, should this reference the helm chart instead of `kustomize`?

I know there's a philosophical debate about this that could be had, but `kustomize` feels more appropriate for local development vs. helm for production.

Reply:
It should be base actually, not local. I'll fix that